metrics_collector/backup_collector.go

```diff
@@ -38,6 +39,16 @@ func NewBackupCollector(
 		Type: prometheus.GaugeValue,
 	}
 
+	bc.sizeTotalMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemBackup, "actual_size_total_bytes"),
+			"Sum of actual size of backups owned by this node",
```

Comment:
Feel this metric is meaningless for us because there is no actual data stored on the node. 

Let's remove this metrics. WDYT?